README.md

```diff
@@ -166,7 +166,34 @@ Users that are frequent contributors and are involved in discussion (join the sl
 ### Stay Scoped
 
 Try to keep your PRs limited to one particular issue, and don't make changes that are out of scope for that issue. If you notice something that needs attention but is out of scope, please [create a new issue](https://github.com/rubyforgood/human-essentials/issues/new).
+## Debugging
+To debug, drop a ``binding.remote_pry`` into the line where you want execution to pause at. Then run ``pry-remote`` in the terminal to connect to it.
+https://github.com/Mon-Ouie/pry-remote
+```
+class Foo
```

Comment:
I don't think we need a full example of how to use `binding.remote_pry`, as the link above shows that.